src/components/Sidebar/Menu.scss

```diff
@@ -0,0 +1,186 @@
+@mixin sidebar {
```

Comment:
don't use sass for this we use material with makeStyles please change this

Reply:
please delete sass, we are using material

Reply:
resolve